CHANGELOG.md

```diff
@@ -4,7 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.3.0] - 2020-11-02
+## [2.4.0] - 2020-11-03
+### Added
+- Added `plex_search` to AND searches together
```

Comment:
I'm confused about this. So if `plex_search` does an AND, how do I do an OR? How would this README example work:
```
collections:
  90s Movies:
    plex_search:
      year:
        - 1990
        - 1991
        - 1992
        - 1993
        - 1994
        - 1995
        - 1996
        - 1997
        - 1998
        - 1999
```

Reply:
it ANDs different attributes and ORs multiples of the same attribute I can try and clarify that in the readme. The program will also tell you exactly how it's searching so for your example it would look like this:

```
| Processing Plex Search: year(1990 OR 1991 OR 1992 OR 1993 OR 1994 OR 1995 OR 1996 OR 1997 OR 1998 OR 1999)
``` 
it will show each AND on the subsequent lines

if you want to AND the same attribute you have to use subfilters

Reply:
Got it. Nice addition to report the actual query.